roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml

```diff
@@ -45,8 +45,29 @@
     body:
       type: "alertmanager"
       name: "{{ cluster_name }}"
+      default_route:
+        slack:
+          channel_id: "{{ slack_id }}"
+          enabled: true
     status_code: [200, 201]
-  when: not integration_exists_for_cluster
+  when: not integration_exists_for_cluster and slack_id is defined
+  register: grafana_integration_response
+  failed_when: false
```

Comment:
setting failed_when to false basically passes the task even when the API request fails. Is this how we want it to be?

Reply:
yes when it fails we should get an error , updating